.devcontainer/Dockerfile

```diff
@@ -15,4 +15,8 @@ RUN \
     library-scripts/provide-user.sh ${USERNAME} ${USER_UID} ${USER_GID} && \
     rm -rf /library-scripts
 
+SHELL ["/bin/bash", "-c"]
+
 USER $USERNAME
+
```

Comment:
The WORKDIR line can be removed, it only affects the commands in the remaining part of the Dockerfile. This made me realize that the USER line should also be removed.